Resources/Prototypes/_Goobstation/Research/experimental.yml

```
@@ -0,0 +1,16 @@
+- type: technology
+  id: ModsuitsTech
+  name: research-technology-modsuits
+  icon:
+    sprite: _Goobstation/Objects/Specific/Robotics/modsuit_parts.rsi
+    state: mod-core-standard
+  discipline: Experimental
+  tier: 1
+  cost: 5000
+  recipeUnlocks:
```

```suggestion
  tier: 3
  cost: 5000
  technologyPrerequisites:
  - AdvancedTacsuits
  softCapContribution: 1.5
  recipeUnlocks:
```
We have some Research tree refactors here on EE, so we can take advantage of that by including Modsuits into the tech trees. As a subtle way to make it so that you can't actually ignore Arsenal, we make the Modsuits require you first research Tacsuits(Which are Arsenal T2).